components/WhatICanHelpYouWith/molecules/WhatICanHelpYouWithUnit.tsx

```diff
@@ -0,0 +1,56 @@
+import { VscArrowRight } from 'react-icons/vsc';
+
+import { LargeParagraph, SmallParagraph } from 'components/atoms/typography.styles';
+import PsychoLink from 'components/atoms/PsychoLink';
+import {
+  Black050, Green100, Green050,
+} from 'components/styling/colors';
+import { WhatICanHelpYouWithItemData } from 'components/WhatICanHelpYouWith/content/whatICanHelpYouWithItems';
+import {
+  WhatICanHelpYouWithItemContainer,
+  WhatICanHelpYouWithIconContainer,
+  LightGreySquare,
```

Comment:
Same idea here. If these are "local" styles related to this component, why do you call one of them `LightGreySquare` and the other is `WhatICanHelpYouWithitemContainer` ? 😄 Why not `WhatICanHelpYouWithLIghtGreySquare` ? Either you are planning to use `LightGreySquare` elsewhere (and then you are having it in the wrong styles), or you are using a naming convention which is inconsistent 😄